test/bultin-addons/core/template-linter-test.ts

```diff
@@ -73,4 +74,46 @@ describe('ProjectTemplateLinter', () => {
       },
     ]);
   });
+
+  test('it return valid result on template with line breaks', async () => {
+    output = '<div class="md:flex p-2 bg-white rounded-lg">\n  <h3 class="mb-4">Hello</h3>\n</div>';
+    const params = {
+      context: {
+        diagnostics: [
+          {
+            range: { start: { line: 0, character: 5 }, end: { line: 0, character: 45 } },
+            message: "HTML class attribute sorting is: 'md:flex bg-white rounded-lg p-2', but should be: 'md:flex p-2 bg-white rounded-lg' (fixable)",
+            severity: 1,
+            code: 'class-order',
+            source: 'ember-template-lint',
+          },
+        ],
+      },
+      textDocument: {
+        uri: 'layout.hbs',
+      },
+      document: {
+        getText: (): string => '<div class="md:flex bg-white rounded-lg p-2">\n  <h3 class="mb-4">Hello</h3>\n</div>',
+      },
+      range: { start: { line: 0, character: 14 }, end: { line: 0, character: 14 } },
+    };
+    const result = await instance.onCodeAction('', params);
+
+    expect(result).toStrictEqual([
+      {
+        edit: {
+          changes: {
+            'layout.hbs': [
+              {
+                range: { start: { line: 0, character: 0 }, end: { line: 2, character: 6 } },
+                newText: '<div class="md:flex p-2 bg-white rounded-lg">\n  <h3 class="mb-4">Hello</h3>\n</div>',
```

Comment:
somewhat unrelated, but is there a rule for 20+ tailwind classes? having all of those on one line is obnoxious, and it'd be great if there was an auto-fixer to multi-line them

Reply:
cc @MichalBryxi

Reply:
I believe that would belong to the template-lint plugin then? https://gitlab.com/michal.bryxi/ember-template-lint-plugin-tailwindcss/-/issues/4